### PR TITLE
Fix int/bigint mismatch between EF and manual SQL scripts

### PR DESCRIPTION
**Pull request by benagain**

### CODE REVIEW

File: src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/TestsConnectionFactory.cs [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        static TestsDbConnectionFactory()
+        {
+            //*
+            factory = new SqliteTestsConnectionFactory();
+            ConnectionString = $"Data Source={AcceptanceTestsData.AcceptanceTestsDatabaseName}";
+            /*/
+            factory = new SqlServerTestsConnectionFactory();
+            ConnectionString = "Data Source=.;Initial Catalog=SFA.DAS.ApprenticeCommitments.AcceptanceTests;Integrated Security=True;";
+            //*/
+        }
```
Comment (Pauljgraham) [inline src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/TestsConnectionFactory.cs:22]: nice

File: src/SFA.DAS.ApprenticeCommitments/Data/Models/ApprenticeCommitmentsDbContext.cs [hunk 2]
```diff
@@ -33,6 +33,7 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                     c =>
                     {
                         c.HasKey("Id");
+                        c.Property(typeof(long), "Id");
```
Comment (Pauljgraham) [inline src/SFA.DAS.ApprenticeCommitments/Data/Models/ApprenticeCommitmentsDbContext.cs:36]: nice, never thought of doing it this way

**Review by Pauljgraham — APPROVED** [review]
Good idea of being able to test against SQL

**Comment by arvinthseran** [comment]
Regression test results : https://dev.azure.com/sfa-gov-uk/Digital%20Apprenticeship%20Service/_releaseProgress?releaseId=55165&_a=release-environment-logs&environmentId=265313

**Pull request merged**